backend/src/main/java/hu/flowacademy/meteo/controller/AirDataController.java

```diff
@@ -0,0 +1,26 @@
+package hu.flowacademy.meteo.controller;
+
+import hu.flowacademy.meteo.dto.AirDataDto;
+import hu.flowacademy.meteo.dto.BetweenDatesDto;
+import hu.flowacademy.meteo.dto.MeasurmentDto;
+import hu.flowacademy.meteo.service.MeasurmentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+public class AirDataController {
+
+    private final MeasurmentService measurmentService;
+
+    @PostMapping("air")
+    public List<AirDataDto> getAllBetweenDates
+            (@RequestBody BetweenDatesDto betweenDatesDto) {
```

Comment:
Ha nagy mennyiségű adat megy vissza (a start-end date alapján akár a teljes megfelelő typusú DB tartalom is mehet), akkor a page kezelést veegyétek fel a listára (akár külön storyba is mehet) Hasznos oldala: https://www.baeldung.com/spring-data-jpa-pagination-sorting ,  https://www.baeldung.com/rest-api-pagination-in-spring.

Reply:
Csak a chart-okba kell tobb adat pont-ot lekerni. Igy a pagination nem az igazi megoldas szerintem, egy limitet adnek inkabb a lekeresekhez, es ha a user tul nagy intervallumot kert le, inkabb warningot dobnek neki. Es mivel szabalyos idkozonket vannak adataink igazabol a frontenden is egyszeru limitalni a valaszthato idointervallumot.